apps/next-product-starter/src/pages/video/[slug].tsx

```diff
@@ -208,20 +240,45 @@ const VideoCuesList: React.FC<any> = () => {
   )
 }
 
-const Page = () => {
+const Page = ({data}: any) => {
   return (
     <Layout>
       <VideoProvider
+        services={{
+          addCueNote,
+          loadViewer:
+            (context: VideoStateContext, _event: VideoEvent) => async () => {
+              context.viewer = VIEWER[1] as any
+            },
+        }}
         actions={{
           onVideoEnded: (_context: any, _event: any) => {
             console.log('do stuff')
           },
         }}
       >
```

Comment:
`addCueNote` is a service passed down from product level which lets us make state transitions based on its result.

`loadViewer` is using hardcoded data at the moment as I needed way to test and demonstrate logic based on `contact_id`